test/integration/services/quotes/sources/quote-sources.spec.ts

```diff
@@ -111,7 +112,7 @@ describe.skip('Quote Sources [External Quotes]', () => {
             buyToken: nativeToken,
             order: {
               type: 'sell',
-              sellAmount: parseUnits('1000', 6),
+              sellAmount: parseUnits('1000', 18),
```

Comment:
We should probably avoid this or all tests will fail in other chains. Maybe we can read the stable token's amount of decimals?